src/seedu/addressbook/data/person/Address.java

```diff
@@ -11,9 +11,22 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
-    public final String value;
+    
+    public static final int ADDRESS_BLOCK_INDEX = 0;
+    public static final int ADDRESS_STREET_INDEX = 1;
+    public static final int ADDRESS_UNIT_INDEX = 2;
+    public static final int ADDRESS_CODE_INDEX = 3;
```

Comment:
Good to see you using these to remove the magic number. BTW, why not using `enum`?